ledger.go

```diff
@@ -238,12 +243,14 @@ func (l *Ledger) Close() {
 // invalid or fails any validation checks, an error is returned. No error
```

Comment:
The comment for this function needs to be updated to indicate that it accepts many transactions and remove references to failing